packages/ui-popover/src/Popover/props.ts

```diff
@@ -38,6 +38,32 @@ import type {
 import type { BidirectionalProps } from '@instructure/ui-i18n'
 import type { PropValidators } from '@instructure/shared-types'
 
+const placementPropValues = [
```

Comment:
Please put a comment above this:

```js
// Copied list from "PositionPropTypes.placement" so that it appears as options in the properties table. 
// TODO: try to use "PositionPropTypes.placement" again once Popover is fully typed and uses @tsProps
```

Reply:
Yes.

Reply:
Cool, thanks!